examples/cloudml-bee-health-detection/trainer/model.py

```diff
@@ -29,6 +29,9 @@
 import tensorflow_hub as hub
 
 
+BEE_SUBSPECIES = ['Carniolan', 'Italian', 'Russian', 'Other']
```

Comment:
I missed this before, but since you're not using numpy anymore you can get rid of that import.

Reply:
will fix